assets/src/edit-story/app/prepublish/guidance/media.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { PAGE_HEIGHT, PAGE_WIDTH } from '../../../constants';
+
+const SAFE_ZONE_AREA = PAGE_HEIGHT * PAGE_WIDTH;
+
+const MIN_PIXEL_DENSITY = 2;
+
+const MAX_VIDEO_LENGTH_SECONDS = 60;
+const MAX_VIDEO_WIDTH = 3840;
+const MAX_VIDEO_HEIGHT = 2160;
+const MIN_VIDEO_HEIGHT = 480;
+const MIN_VIDEO_WIDTH = 852;
+
+// todo: get video frames per second
```

Comment:
https://media3p.googleapis.com/v1/media?filter=provider%3Acoverr&page_size=20&key=[token] doesn't provide this data

Reply:
I'll see if this information is available from the Coverr API...

Reply:
Doesn't look like it: https://api.coverr.co/docs/videos/#response